cmd/utils.go

```diff
@@ -157,3 +162,69 @@ func getLookupType(l string) minio.BucketLookupType {
 	}
 	return minio.BucketLookupAuto
 }
+
+// sumMD5Base64 calculate md5sum for an input byte array, returns base64 encoded.
+func sumMD5Base64(data []byte) string {
+	hash := md5.New()
+	hash.Write(data)
+	return base64.StdEncoding.EncodeToString(hash.Sum(nil))
+}
+
+// struct representing object prefix and sse keys association.
+type prefixSSEPair struct {
+	prefix string
+	sseKey string
+}
+
+// parse list of space separated alias/prefix=sse key values entered on command line and
+// construct a map of alias to prefix and sse pairs
+func parseEncryptionKeys(ssekeys string) (encMap map[string][]prefixSSEPair, err *probe.Error) {
+	encMap = make(map[string][]prefixSSEPair)
+	if ssekeys == "" {
+		return
+	}
+	fields := strings.Fields(ssekeys)
+	for _, field := range fields {
+		r := regexp.MustCompile("([^=\\s]+)=([^\\s]+)")
```

Comment:
This can be initialized only one time outside the for loop

Reply:
done.